Scenes/CharacterCreation/CharacterCreation.gd

```diff
@@ -18,33 +16,45 @@ func _ready():
+		profilePanel.get_node("Pic").flip_h = CharacterDefaults.flip_profile[profile][0]
+		profilePanel.get_node("Pic").flip_v = CharacterDefaults.flip_profile[profile][1]
+	var spec = get_node("MainMenu/Choices/Stats/Display/Menu/SPEC").get_children()
+	spec[0].get_node("Labels/Label").text = "Strength"
+	spec[1].get_node("Labels/Label").text = "Perception"
+	spec[2].get_node("Labels/Label").text = "Endurace"
+	spec[3].get_node("Labels/Label").text = "Charisma"
+	
+	var ial = get_node("MainMenu/Choices/Stats/Display/Menu/IAL").get_children()
+	ial[0].get_node("Labels/Label").text = "Intelligence"
+	ial[1].get_node("Labels/Label").text = "Agility"
+	ial[2].get_node("Labels/Label").text = "Luck"
 #	get_node("MainMenu/Choices/ProfileSelection/Profiles").get_children()[selectedCharacter].get_node("Border").show()
 #	get_node("MainMenu/Choices/ProfileSelection/Profiles").get_children()[selectedCharacter].chosen = true
-	get_node("MainMenu/Choices/Equipment/Classes/Knight").get_node("ClassName").text = "Knight"
-	get_node("MainMenu/Choices/Equipment/Classes/Knight").get_node("ImgCenter/ClassImg").texture = load("res://Assets/Images/Icons/Classes/Badge_warrior.png")
-	get_node("MainMenu/Choices/Equipment/Classes/Berserker").get_node("ClassName").text = "Berserker"
-	get_node("MainMenu/Choices/Equipment/Classes/Berserker").get_node("ImgCenter/ClassImg").texture = load("res://Assets/Images/Icons/Classes/Badge_barbarian.png")
-	get_node("MainMenu/Choices/Equipment/Classes/BattleMage").get_node("ClassName").text = "Battle Mage"
-	get_node("MainMenu/Choices/Equipment/Classes/BattleMage").get_node("ImgCenter/ClassImg").texture = load("res://Assets/Images/Icons/Classes/Badge_mage.png")
-	get_node("MainMenu/Choices/Equipment/Classes/QuickShooter").get_node("ClassName").text = "Quick Shooter"
-	get_node("MainMenu/Choices/Equipment/Classes/QuickShooter").get_node("ImgCenter/ClassImg").texture = load("res://Assets/Images/Icons/Classes/Badge_hunter.PNG")
+	var knight = get_node("MainMenu/Choices/Equipment/Classes/Knight")
+	knight.get_node("ClassName").text = "Knight"
+	knight.get_node("ImgCenter/ClassImg").texture = load("res://Assets/Images/Icons/Classes/Badge_warrior.png")
+	
+	var berserker = get_node("MainMenu/Choices/Equipment/Classes/Berserker")
+	berserker.get_node("ClassName").text = "Berserker"
+	berserker.get_node("ImgCenter/ClassImg").texture = load("res://Assets/Images/Icons/Classes/Badge_barbarian.png")
+	
+	var battle_mage = get_node("MainMenu/Choices/Equipment/Classes/BattleMage")
+	battle_mage.get_node("ClassName").text = "Battle Mage"
+	battle_mage.get_node("ImgCenter/ClassImg").texture = load("res://Assets/Images/Icons/Classes/Badge_mage.png")
+	
+	var quick_shooter = get_node("MainMenu/Choices/Equipment/Classes/QuickShooter")
+	quick_shooter.get_node("ClassName").text = "Quick Shooter"
+	quick_shooter.get_node("ImgCenter/ClassImg").texture = load("res://Assets/Images/Icons/Classes/Badge_hunter.PNG")
```

Comment:
What was changed here?
Just making the dynamic fields pre named?

Reply:
Yep! Refactoring work